packages/babel-parser/src/plugins/jsx/index.js

```diff
@@ -107,6 +107,20 @@ export default (superClass: Class<Parser>): Class<Parser> =>
             chunkStart = this.state.pos;
             break;
 
+          case charCodes.greaterThan:
+          case charCodes.rightCurlyBrace: {
+            const htmlEntity =
+              ch === charCodes.rightCurlyBrace ? "&rbrace;" : "&gt;";
+            throw this.raise(
+              this.state.pos,
+              `Unexpected token \`${
+                this.input[this.state.pos]
+              }\`. Did you mean \`${htmlEntity}\`, or \`{'${
```

Comment:
```suggestion
              }\`. Did you mean \`${htmlEntity}\` or \`{'${
```

Small thing I just noticed: Can we get rid of this extra comma in the error message?